omod_1.x/src/main/java/org/openmrs/module/openhmis/cashier/web/controller/JasperReportController.java

```diff
@@ -61,9 +62,8 @@ public String render(@RequestParam(value = "reportId", required = true) int repo
 		try {
 			ReportGenerator.generate(report, params, false, true);
 		} catch (IOException e) {
-			response.sendError(HttpServletResponse.SC_INTERNAL_SERVER_ERROR, "Error generating cashier shift report for " +
-					"timesheet '" + temp + "'");
-			return null;
+			message = "Error generating cashier shift report for " + "timesheet '" + temp + "'";
```

Comment:
Include the exception message in the message that gets displayed on the report error page.